app/config/locale/translations/no.json

```
@@ -3,30 +3,30 @@
     "settings.locale": "no",
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Kontobekreftelse",
+    "emails.verification.hello": "Hallo {{name}}",
+    "emails.verification.body": "Følg denne lengken for å bekrefte din e-post adresse",
+    "emails.verification.footer": "Dersom du ikke ba om å bekrefte e-post adressen, kan du se bort fra denne meldingen.",
+    "emails.verification.thanks": "Takk",
+    "emails.verification.signature": "{{project}} team",
+    "emails.magicSession.subject": "Pålogging",
+    "emails.magicSession.hello": "Hallo",
```

```suggestion
    "emails.magicSession.hello": "Hallo,",
```
Please follow the punctuation from `en.json`

---

Oops, small typo, I'll fix that right away!

---

I changed it to "Hei," which more closely follows the equivalent line in `en.json`

---

You can resolve it yourself if you feel the issue has been addressed

---

Also please check the punctuation in all the other string as well. Not just line 13

---

Went over everything and found another period that had eluded me. Also fixed a typo. The changes I made follow the same punctuation as in the English version as far as I can see, so I'm resolving it now. :)